bastionzero/zli
# Add `create` functionality to `zli policy`

## Description of the change

This PR adds `create` functionality to the `zli policy` command. 
`zli policy create-recording` -> creates a new session recording policy
`zli policy create-proxy` -> creates a new proxy policy
`zli policy create-tconnect` -> creates a new target connect policy
`zli policy create-cluster` -> creates a new kubernetes cluster policy

## Testing

**Manual tests:**
You can download two bash scripts I wrote in our `Engineering` shared drive:
[Create Policy Test Scripts](https://drive.google.com/drive/folders/1tyLKJrAr9tavKn2Yns9XXNwlZbDCATj7?usp=sharing)

These scripts should be run against **staging** where everything is already set up for them. You should be using your local instance of zli for this. In both scripts, `success-test-cases.sh` and  `error-test-cases.sh`, I made clear comments about what each test case is aiming for. If there's any cases I'm missing that you think would be beneficial to see please let me know.

**System tests:**
These commands are covered by the `policySuite()` test suite in the system tests.

**backend branch:** 
**bzero branch:** develop

#### Ready to run system tests?


## Relevant release note information

Release Notes: Users can now create a policy from the zli

## Related JIRA tickets

Relates to JIRA: CWC-1541

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?


If yes, please explain:
